services/audio/src/main/java/ca/bc/gov/tno/services/audio/config/AudioConfig.java

```diff
@@ -3,54 +3,61 @@
 import org.springframework.context.annotation.Configuration;
 
 import ca.bc.gov.tno.dal.db.entities.DataSource;
+import ca.bc.gov.tno.dal.db.services.interfaces.IDataSourceService;
 import ca.bc.gov.tno.services.data.config.DataSourceConfig;
 
 /**
  * Configuration settings for the default Audio Feed. These values will be
  * used if a connection to the database cannot be made.
  */
 @Configuration
-//@ConfigurationProperties("data")
 public class AudioConfig extends DataSourceConfig {
+
   /**
-   * URL to the audio streaming source
+   * Directory in which capturedAudio is being stored
    */
-  private String audioUrl;
+  private String captureDir;
 
   /**
-   * Capture command template
+   * Clip command template
    */
-  private String captureCmd;
+  private String clipCmd;
 
   /**
-   * Directory in which to store captured audio
+   * Clip directory
    */
-  private String captureDir;
+  private String clipDir;
 
   /**
-   * Clip command template
+   * Time at which the audio began streaming
    */
-  private String clipCmd;
+  private long streamStartTime;
 
   /**
-   * Clip duration
+   * Code (name) of the capture service linked to this clip service.
    */
-  private String clipDuration;
+  private String captureService;
 
   /**
-   * Clip directory
+   * Code (name) of this audio service
    */
-  private String clipDir;
+  private String audioService;
 
   /**
-   * Time at which the audio began streaming
+   * Was the last clip command validated and extracted?
    */
-  private long streamStartTime;
+  private boolean clipSuccess;
```

Comment:
clipSuccess indicates that the clip was extracted successfully. At this point (because we're mostly following the happy path) the only check I'm doing is that the audio to be clipped does exist in the capture file. This is determined by ensuring the start time of the clip is after the capture process began recording. clipSuccess is used in KafkaAudioProducer.send() to avoid adding a clip entry, in Kafka, for a clip that wasn't created.

Reply:
Seems odd it would want to or try to send a message to kafka if the clip was unsuccessful.  This appears to be a process flow issue, as it shouldn't even try if it was unsuccessful.